service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -5,27 +5,102 @@
 namespace Application\Controller;
 
 use Application\Contracts\OpgApiServiceInterface;
+use Application\Exceptions\OpgApiException;
 use Application\Forms\DrivingLicenceNumber;
+use Application\Forms\IdQuestions;
 use Application\Forms\PassportNumber;
 use Application\Forms\PassportDate;
+use Application\Services\SiriusApiService;
+use Laminas\Http\Response;
 use Laminas\Mvc\Controller\AbstractActionController;
+use Laminas\Mvc\Controller\Plugin\Redirect;
 use Laminas\View\Model\ViewModel;
-use Application\Forms\NationalInsuranceNumber;
 use Laminas\Form\Annotation\AttributeBuilder;
+use Application\Forms\NationalInsuranceNumber;
 
 class IndexController extends AbstractActionController
 {
     protected $plugins;
-
-    public function __construct(private readonly OpgApiServiceInterface $opgApiService)
-    {
+    public function __construct(
+        private readonly OpgApiServiceInterface $opgApiService,
+        private readonly SiriusApiService $siriusApiService,
+    ) {
     }
 
     public function indexAction()
     {
         return new ViewModel();
     }
 
+    public function startAction(): ViewModel
+    {
+        $lpas = [];
+        foreach ($this->params()->fromQuery("lpas") as $lpaUid) {
+            $data = $this->siriusApiService->getLpaByUid($lpaUid, $this->getRequest());
+            $lpas[] = $data['opg.poas.lpastore'];
+        }
+
+        $detailsData = $this->opgApiService->getDetailsData();
+
+        // Find the details of the actor (donor or certificate provider, based on URL) that we need to ID check them
+
+        // Create a case in the API with the LPA UID and the actors' details
+
+        // Redirect to the "select which ID to use" page for this case
+
+        $case = '49895f88-501b-4491-8381-e8aeeaef177d';
+
+        $view = new ViewModel([
+            'lpaUids' => $this->params()->fromQuery("lpas"),
+            'type' => $this->params()->fromQuery("personType"),
+            'lpas' => $lpas,
+            'case' => $case,
+            'details' => $detailsData,
+        ]);
+
+        return $view->setTemplate('application/pages/start');
+    }
+
+    public function howWillDonorConfirmAction(): ViewModel
+    {
+        $uuid = $this->params()->fromRoute("uuid");
+
+        if (count($this->getRequest()->getPost())) {
+            $formData = $this->getRequest()->getPost()->toArray();
+
+            switch ($formData['id_method']) {
+                case 'Passport':
```

Comment:
Probably one for a future PR, but I think there's some tidying up we can do here since it's basically a magic string: the frontend code is relying on the API having a particular output. Since this is only used for routing requests, I think we can ditch the API endpoint and just define the "ID options" in the service-front code.

Reply:
I've [made a ticket for this](https://opgtransform.atlassian.net/browse/ID-93)